### DISCUSSION TITLE
Need Projects integration on mobile app

### DISCUSSION THREAD

**Discussion by g7i** [community/community#18395] [Mobile]
Need Projects integration on mobile app just like we have on website

**Comment by eliperkins** [comment]
Hey @g7i, thanks for your feedback about this.

Our team is hard at work on bringing the new Issues/Projects experience to GitHub Mobile. Stay tuned for more updates to come!

In the meantime, you can utilize GitHub Projects within GitHub Mobile by going to the ℹ️ button on an issue or pull request, and scrolling to the "Projects" section to see information about the project that the issue or pull request belongs to.

If there are specific features of GitHub Issues or GitHub Projects that you'd love to see on mobile, please let us know!
